styles/aboutus.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    --background-color: #FAF4E8;
+    --feature-color: #34959E;
+    --footer-color: #9D6034;
+    --nav-bar: #DAA46D;
+    --text-color: #382B22;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    background-color: var(--background-color);
+    font-family:Arial, Helvetica, sans-serif;
+    height: 90%;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+h2 {
+    margin: 0;
+}
+
+h3 {
+    margin: 0;
+}
+p {
+    margin: 0;
+}
+
+.header {
+    
```

Comment:
Header element should not get any CSS in here since the header is a globally used in all html files. 
Which means the Header css should only be in global.css.